parachains/runtimes/assets/westmint/src/xcm_config.rs

```diff
@@ -205,6 +243,34 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 				pallet_assets::Call::touch { .. } |
 				pallet_assets::Call::refund { .. },
 			) |
+			RuntimeCall::ForeignAssets(
+				/* avoided: mint, burn */
```

Comment:
why avoid here? `mint` and `burn` permissions should never be available to any account anyway, right?